src/index.ts

```diff
@@ -88,7 +88,7 @@ interface ElysiaYogaConfig<
  */
 export const yoga =
     <
-        const TypeDefs extends string,
+        const TypeDefs extends string | TypeSource,
```

Comment:
I spotted that as well on my issue pr: #8 , TypeSource already has string support so you should be fine with just
```suggestion
        const TypeDefs extends TypeSource,
```